js/scripts.js

```diff
@@ -35,7 +35,7 @@ var MoviesList = React.createClass({
   },
 
   render: function () {
-    const itemMovie = this.props.moviesList.map(function (item) {
+    const itemMovie = this.props.moviesList.maps(function (item) {
```

Comment:
`map` is a method, so should be `map`

i was thinking about variable name `itemMovie` - this should be in plural